SQL/0000-00-00-schema.sql

```diff
@@ -2343,3 +2343,29 @@ CREATE TABLE `CNV` (
   FOREIGN KEY (`GenomeLocID`) REFERENCES genome_loc(`GenomeLocID`),
   FOREIGN KEY (`CandID`) REFERENCES candidate(`CandID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+CREATE TABLE `log` (
+  `LogID` int(11) NOT NULL AUTO_INCREMENT,
+  `LogTypeID` int(11) NOT NULL DEFAULT '0',
+  `ProcessID` int(11) NOT NULL DEFAULT '0',
+  `Date_Created` datetime NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `Message` text,
+  `Error` tinyint(1) DEFAULT '0',
```

Comment:
This should be an enum instead of boolean, because boolean/tinyint(1) gets QuickForm confused by "0" with tinyint in the frontend.